crates/revm/src/db/states/reverts.rs

```diff
@@ -171,6 +171,12 @@ impl AccountRevert {
             wipe_storage: true,
         }
     }
+
+    /// Returns `true` if there is nothing to revert,
+    /// by checking that account info and storage have been left untouched
+    pub fn is_empty(&self) -> bool {
+        self.account == AccountInfoRevert::DoNothing && self.storage.is_empty()
```

Comment:
Just remembered one more thing. check if `wipe_storage` is `false`, if it is `true` it is an action that needs to happen.

Reply:
Yeah I had that doubt but I forgot to mention it. Tomorrow I’ll fix all comments. Thanks!